### PR TITLE
Editing

### PR DESCRIPTION
**Pull request by Buzzardo**
In two commits, I have made various changes to spring-batch-into. I made the references to Accenture and SpringSource be historical in nature. I also gave it an "English teacher" pass. Finally, I had some questions and issues, which I marked with TODO comments.

### CODE REVIEW

File: spring-batch-docs/asciidoc/spring-batch-intro.adoc [hunk 1]
```diff
@@ -292,176 +304,176 @@ Batch applications can additionally be categorized by their input source:
...
+Preprocessing could be a large overhead, as it has to calculate and determine the bounds of each portion of the record set.
 
-_2. Breakup by a Key Column_
+_2. Break up by a Key Column_
 
-This involves breaking up the input record set by a key column such as a location code, and assigning data from each key to a batch instance. In order to achieve this, column values can either be
+This involves breaking up the input record set by a key column, such as a location code, and assigning data from each key to a batch instance. In order to achieve this, column values can be either:
 
-_3. Assigned to a batch instance via a partitioning table (see below for details)._
+_* Assigned to a batch instance by a partitioning table (described later in this section)._
```
Comment (mminella) [inline spring-batch-docs/asciidoc/spring-batch-intro.adoc]: Is this supposed to be just an * in the rendered output?  It looks odd that it's not a bullet or something...

File: spring-batch-docs/asciidoc/spring-batch-intro.adoc [hunk 2]
```diff
@@ -292,176 +304,176 @@ Batch applications can additionally be categorized by their input source:
...
 
-_4. Assigned to a batch instance by a portion of the value (e.g. values 0000-0999, 1000 - 1999, etc.)_
+_* Assigned to a batch instance by a portion of the value (such as 0000-0999, 1000 - 1999, and so on)._
```
Comment (mminella) [inline spring-batch-docs/asciidoc/spring-batch-intro.adoc]: Is this supposed to be just an * in the rendered output?  It looks odd that it's not a bullet or something...

File: spring-batch-docs/asciidoc/spring-batch-intro.adoc (hunk 1 above)
Comment (Buzzardo) [reply spring-batch-docs/asciidoc/spring-batch-intro.adoc]: I turned 3 and 4 into bullets under 2. That's why the both start with a lone asterisk.

File: spring-batch-docs/asciidoc/spring-batch-intro.adoc (hunk 2 above)
Comment (Buzzardo) [reply spring-batch-docs/asciidoc/spring-batch-intro.adoc]: I turned 3 and 4 into bullets under 2. That's why the both start with a lone asterisk.

**Comment by mminella** [comment]
LGTM.  Squashed and merged as de5792b0f26f4a885f43b316f693e38a1fbfcb3e

**Pull request closed without merging**